mfem/ginkgo/CMakeLists.txt

```
@@ -1,8 +1,10 @@
-add_executable(mfem_ex1_gko mfem_ex1_gko.cpp)
-target_sources(mfem_ex1_gko PRIVATE mfem_wrapper.cpp)
+if (ENABLE_HIP)
```

For CUDA we need this:
```suggestion
if (ENABLE_CUDA)
    set_source_files_properties(mfem_ex22_gko.cpp PROPERTIES LANGUAGE CUDA)
endif()
if (ENABLE_HIP)
```